### ISSUE TITLE
New physical button

### ISSUE THREAD

**Issue by hevv88** [dadav/pwnagotchi#16]
I saw you try to update/upgrade as much as you can and maybe my idea is not new, but is there a chance to be added a physical power on/off button who will work without adding nothing after a fresh installation

**Comment by qkum** [comment]
Bro, you can't add a physical button by writing code :) 

If he could write some code that would work with a button? I think the easy solution is you on amazon buying a button that don't need software to turn it on/off. 

From what I know all physical on/off buttons needs a script to be added to the pi to shut it down in a save way so the sd card don't corrupt. 

There is no easy fix my friend. 

The easy fix is to become a pro and always be in connection with the pi so you can turn it on/off through ssh/terminal or the lovely pwnagotchi UI at 192.168.1.xxx:8080 (pwnagotchi ip + :8080) 👍 

You just gotta learn to live without that on/off button my friend :) 

I know how anything it is when you are new to this like I was 2 years ago. 

The good thing is it forces you to learn if you want to be part of this space.

The learning curve is huge but you will be handsomely paid and have fun at work for the rest of your life if you never give up like me 🚀

**Comment by RChadwick7** [comment]
Actually, I found it relatively easy to add a physical button to the attached UPS Lite. I simply soldered the button to two pins. The UPS Lite isn't required, and the button could just be soldered to the Pi0W. I configured one of the plugins to shut it down. It would have been nice if Pwhagotchi would do this 'Out of the box', but it's not hard to add. What the Pwnagotchi really needs is a shutdown plugin, which really should be easy to implement, but someone overthought things. I'm adding a hardware part to trigger the shutdown line, just need to get to it someday.
![20210320_221530](https://user-images.githubusercontent.com/8503659/111891711-35dd6400-89cb-11eb-90ac-c00553f0bc61.jpg)
